### DISCUSSION TITLE
build error in IntelliJ IDEA 2023.3.1

### DISCUSSION THREAD

**Discussion by tianshaokai** [serbelga/Todometer-KMP#448] [Q&A] [answered]
<img width="1486" alt="image" src="https://github.com/serbelga/Todometer-KMP/assets/6706496/12530ee9-431d-4a48-a37e-20ca19288ea5">

What was the author using to build composite for the first time learning it.

**Comment by serbelga** [comment, marked as answer]
You can use Android Studio Hedgehog | 2023.1.1 Patch 1 https://androidstudio.googleblog.com/2024/01/android-studio-hedgehog-202311-patch-1.html
or Android Studio Iguana | 2023.2.1 Beta 1
https://androidstudio.googleblog.com/2023/12/android-studio-iguana-202321-beta-1-now.html
